docs/configuration.rst

```diff
@@ -48,3 +48,24 @@ Environment variables
 
 ``SECRET_KEY``
   Controls `Django's secret key <https://docs.djangoproject.com/en/1.10/ref/settings/#secret-key>`_ setting.
+
+``ENABLE_ELASTICSEARCH``
```

Comment:
Why do we need a separate variable for that? Why would I set `ELASTICSEARCH_URL` and `ELASTICSEARCH_INDEX_NAME` if I did not intend to use Elasticsearch?